bundle-workflow/README.md

```diff
@@ -102,10 +102,8 @@ Tests the OpenSearch bundle.
 
 This workflow contains integration, backwards compatibility and performance tests.
 
-The following example kicks off all test suites for a distribution of OpenSearch 1.1.0.
-
 ```bash
-./bundle-workflow/test.sh manifests/opensearch-1.1.0.yml
+./bundle-workflow/test.sh integ-test
```

Comment:
Which manifest?

Reply:
Not sure if the question was what manifest do we use for integ tests. 
I would like to have #456 update this readme to add the rest of the new parameters to this readme.